src/azure-cli/azure/cli/command_modules/botservice/_params.py

```diff
@@ -106,6 +105,7 @@ def load_arguments(self, _):
                    arg_group='Bot Analytics/Application Insights',
                    help='Azure Application Insights Application ID used to read bot analytics data. Provide an Id if '
                         'you want to view analytics about your bot in the Analytics blade.')
+        c.argument('icon_url', help='Icon URL for bot avatar. 30K max, png only')
```

Comment:
"30K max, png only."
I suggest a formal expression.

Reply:
Done in #11318